packages/gatsby-plugin-image/README.md

```diff
@@ -1,19 +1,25 @@
 # gatsby-plugin-image (beta)
 
-This plugin is a replacement for gatsby-image. It adds [static images](#staticimage), and a [new higher-performance gatsby-image component](#gatsbyimage). It also adds [a new GraphQL resolver](#graphql-resolver) to gatsby-transformer-sharp.
+_The new Gatsby Image plugin is currently in beta, but you can try it out now and see what it can do for the performance of your site._
+
+Adding responsive images to your site while maintaining high performance scores can be difficult to do manually. The Gatsby Image plugin handles the hard parts of producing images in multiple sizes and formats for you!
+
+For full documentation on all configuration options, see [the reference guide](https://www.gatsbyjs.com/docs/reference/)
 
 ## Contents
 
-- [StaticImage](#staticimage) - the new static image component
-- [GatsbyImage](#gatsbyimage) - a high-performance gatsby-image component
-- [gatsbyImageData](#graphql-resolver) - a simpler GraphQL API
+- [Installation](#installation)
+- [Using the Gatsby Image components](#using-the-gatsby-image-components)
+  - [Static images](#static-images)
+  - [Dynamic images](#dynamic-images)
+- [Migrating to gatsby-plugin-image](#migrating)
 
-## Usage
+## Installation
 
-1. Install `gatsby-plugin-image` and `gatsby-plugin-sharp`:
+1. Install `gatsby-plugin-image`, `gatsby-source-filesystem` and `gatsby-plugin-sharp`:
```

Comment:
Should we have a switch case where source-filesystem is for static and transformer is for gatsbyImage? You won't always need it, right?

Reply:
No, not always it's true, but it's not as simple as 1:1 though